### DISCUSSION TITLE
Sql server jdbc name in \plugis directory od the product installation

### DISCUSSION THREAD

**Discussion by amisano** [dbeaver/dbeaver#13848] [Q&A] [answered]
good morning
This is not an issue is only an informations because I have some other problems
What is the name of the .jar file used to connect to a sql server  2008 R2 ?
The exact name of the file in the directory C:\Program Files\DBeaver\plugins
Many and many thanks

**Comment by uslss** [comment, marked as answer]
You could find which driver you use in Connection Properties - Edit Driver Settings - Libraries tab 
![image](https://user-images.githubusercontent.com/31996417/131996892-0a8773f5-79ea-4a8a-b63f-c3c696d7ecc0.png)
